IntegrationTests/README.md

```diff
@@ -0,0 +1,3 @@
+# Integration Tests
+
+Some API calls need more in-depth testing to assure correctness.  For these we have integration tests which are to be executed locally with secrets provided in configuration, primarily through [`dotnet user-secrets`](https://learn.microsoft.com/en-us/aspnet/core/security/app-secrets).
```

Comment:
Add sample commands necessary to add secrets.

Reply:
Done